_data/events.yaml

```diff
@@ -0,0 +1,96 @@
+title: Events - vulekamali
```

Comment:
We can delete this now that there are fixtures, right?

Reply:
Yes! deleted.